master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -736,7 +737,7 @@ private[celeborn] class Master(
     context.reply(CheckQuotaResponse(isAvailable, reason))
   }
 
-  private def workersNotBlacklisted(
+  private def workersAvailable(
```

Comment:
Probably workersAvailable need consider shutdown workers every time? so filter with shutdownWorkerSnapshot together with blacklist, then param  may for other Scenario.

Reply:
Yea, done